src/robusta/core/sinks/sink_base.py

```diff
@@ -17,6 +19,21 @@ def __init__(self, sink_params: SinkBaseParams, registry):
         self.cluster_name: str = global_config.get("cluster_name", "")
         self.signing_key = global_config.get("signing_key", "")
 
+        self.time_slice = self._build_time_slice_from_params(self.params.activity)
+
+    def _build_time_slice_from_params(self, params: ActivityParams):
+        if params is None:
+            return TimeSliceAlways()
+        else:
+            timezone = params.timezone
+            time_slice = self._interval_to_time_slice(timezone, params.intervals[0])
```

Comment:
Would it be easier to just have an array of `time_slice`, and then, the `Sink` is_active is just:
`any([slice.is_active_now() for slice in time_slice])`

If we do this, we remove the need for `TimeSliceUnion`, which kinda not explicit (both overriding the `+` operator, and the `is_active_now`)

WDYT?

Reply:
That was a good idea. Simplified the code quite a bit.